model/src/main/java/com/basistech/rosette/dm/AnnotatedText.java

```diff
@@ -47,18 +48,19 @@
  * For Json, and Java, the 'adm-json' module provides the supported serialization.
  */
 @SuppressWarnings("deprecation")
-public class AnnotatedText {
+public class AnnotatedText implements Serializable {
+    private static final long serialVersionUID = 222L;
     private final CharSequence data;
     /* The attributes for this text, indexed by type.
      * Only one attribute of a type is permitted, thus the concept
      * of a ListAttribute.
      */
     private final Map<String, BaseAttribute> attributes;
     private final Map<String, List<String>> documentMetadata;
-    private boolean compatMentionsProcessed;
-    private ListAttribute<EntityMention> compatMentions;
-    private boolean compatResolvedEntitiesProcessed;
-    private ListAttribute<ResolvedEntity> compatResolvedEntities;
+    private transient boolean compatMentionsProcessed;
```

Comment:
Transient (and these fields themselves) will go away when we remove the deprecated classes, yes?